contrib/zpages/src/main/java/io/opencensus/zpages/ZPageHttpHandler.java

```diff
@@ -34,52 +32,29 @@
 import java.util.Map;
 
 /**
- * An {@link HttpHandler} that displays information about active and sampled spans recorded using
- * the OpenCensus library.
- *
- * <p>Example usage with {@link HttpServer}:
- *
- * <pre>{@code
- * public class Main {
- *   public static void main(String[] args) throws Exception {
- *     HttpServer server = HttpServer.create(new InetSocketAddress(8000), 10);
- *     TracezHttpHandler.register(server);
- *     server.start();
- *   }
- * }
- * }</pre>
+ * An {@link HttpHandler} that can be used to render HTML pages using any {@code
+ * PageFormatterInterface}.
  */
-public final class TracezHttpHandler implements HttpHandler {
+final class ZPageHttpHandler<T extends PageFormatter> implements HttpHandler {
```

Comment:
Will the type variable be needed for anything besides the `pageFormatter` field?  It looks like `pageFormatter` could be changed to type PageFormatter, and T could be removed.